06-building-a-program-start-to-finish/writing-tests/Output/test/spec/test.js

```diff
@@ -0,0 +1,83 @@
+const chai = require('chai');
+const expect = chai.expect;
+ 
+ const InvertedIndex = require('../../src/js/inverted-index.js');
+ const books = require('../books.json');
+ 
+ const emptyBook = [];
+ const newIndex = new InvertedIndex();
+ const sampleData = ' Mary had a    little #$%^6 lamb, a little lamb Mary had';
+ newIndex.createIndex('books.json', books);
+ 
+ describe('Validate files', () => {
+   it('should check that uploaded file is valid JSON', () => {
```

Comment:
This suite does not actually test for functionality of your Inverted Index implementation.

You should've had a method that does validation somewhere which you'll be testing here.

Also, the fact that the file content is an Array does not mean it's a valid JSON. Kindly checkout `JSON.parse`.

I expect the function that does validation to ensure the correct JSON format is passed also and that should be tested here too.